<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <bold>false</bold>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>695</width>
         <height>544</height>
        </rect>
       </property>
       <widget class="QTextBrowser" name="dispayMessageTextBrowser">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>0</y>
          <width>401</width>
          <height>471</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="inputMessageTextEdit">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>470</y>
          <width>401</width>
          <height>71</height>
         </rect>
        </property>
       </widget>
       <widget class="QScrollBar" name="verticalScrollBar">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>0</y>
          <width>16</width>
          <height>541</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="sendMessagePushButton">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>510</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="findUserLineEdit">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>281</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="searchUserPushButton">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>10</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
       <widget class="QWidget" name="widget_1" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>281</width>
          <height>501</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>715</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>MainMenu</string>
    </property>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
